packages/core/admin/admin/src/content-manager/pages/EditView/Header/index.js

```diff
@@ -189,15 +197,74 @@ const Header = ({
               <Button onClick={toggleWarningUnpublish} variant="tertiary">
                 {formatMessage({
                   id: 'components.popUpWarning.button.cancel',
-                  defaultMessage: 'No, cancel',
+                  defaultMessage: 'Cancel',
                 })}
               </Button>
             }
             endAction={
               <Button variant="danger-light" onClick={handleUnpublish}>
                 {formatMessage({
                   id: 'components.popUpWarning.button.confirm',
-                  defaultMessage: 'Yes, confirm',
+                  defaultMessage: 'Confirm',
+                })}
+              </Button>
+            }
+          />
+        </Dialog>
+      )}
+      {showPublishConfirmation && (
+        <Dialog
```

Comment:
`Dialog` you don't need to conditionally render, `isOpen === false` will mean it renders `null`.